packages/page-coretime/src/utils/index.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright 2017-2024 @polkadot/app-coretime authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { CoretimeInformation } from '@polkadot/react-hooks/types';
+import type { ChainName, RegionInfo } from '../types.js';
+
+import { CoreTimeChainConsts, CoreTimeConsts } from '@polkadot/react-hooks/types';
+import { BN } from '@polkadot/util';
+
+type FirstCycleStartType = Record<
```

Comment:
Start of the cycle, aka first sale initialization, starting with the interlude! Sale starts after the interlude.